ChangeLog.md

```diff
@@ -27,6 +27,9 @@ See docs/process.md for more on how version tagging works.
 - emcc will now treat `.bc` files as source files.  These means that will get
   compiled by clang before being passed to the linker.  This matches the
   behaviour of clang. (#20922)
+- Emscripten now only supports browsers going back to certain versions. The
+  current set of minimum versions are: Chrome 32, Firefox 34, Safari 9 and Node
+  10.19.0. (#20924)
```

Comment:
Maybe add a comment here, "all of which were released at least X years ago (i.e. are very old)", otherwise the casual reader might think "oh no I need to check if any of those are recent".